molo/commenting/tests/test_models.py

```diff
@@ -32,3 +39,10 @@ def test_parent(self):
         second_comment.save()
         [child] = first_comment.children.all()
         self.assertEqual(child, second_comment)
+
+    def test_auto_remove(self):
+        comment = self.mk_comment('first comment')
+        comment.save()
+        comment_flag = self.mk_comment_flag(comment)
+        comment_flag.save()
+        self.assertTrue(comment.is_removed)
```

Comment:
@KaitCrawford You'll need to "refresh" the `comment` since it's update in the DB but not in your current thread. So the easiest way to do that would be to do the following:
```
comment = Comment.objects.get(pk=comment.pk)
```